src/pages/rest/tutorials/inventory/create-shipment.md

```diff
@@ -22,6 +22,10 @@ When you complete a partial or full shipment, Adobe Commerce deducts the reserve
 
 Although you can use the `POST V1/shipment` endpoint to create a shipment, the `POST /V1/order/:orderId/ship` endpoint is a better option in that it is more efficient and the payload can be simpler.
 
+<InlineAlert variant="notice" slots="text"/>
+
+To avoid multiple (duplicate) shipment records creation for the same item in the order, you must use `POST /V1/order/:orderId/ship` endpoint instead of `POST V1/shipment` endpoint. Only the newer `POST /V1/order/:orderId/ship` endpoint can validate total shipped quantity for each item in the order.
```

Comment:
```suggestion
To avoid multiple (duplicate) shipment record creation for the same item in the order, you must use the `POST /V1/order/:orderId/ship` endpoint instead of the `POST V1/shipment` endpoint. Only the newer `POST /V1/order/:orderId/ship` endpoint can validate total shipped quantity for each item in the order.
```